### DISCUSSION TITLE
Implementation of Switch task type in conductor.

### DISCUSSION THREAD

**Discussion by kalyann-vadada** [Netflix/conductor#3634] [Q&A]
Hi,

I can't find any kind of decision making/evaluation expression in the implementation of Switch task type in conductor repository. I have cloned the code from latest main branch. 

https://github.com/Netflix/conductor/blob/main/core/src/main/java/com/netflix/conductor/core/execution/tasks/Switch.java

Please let me know if anything needs to be configured.

**Comment by v1r3n** [comment]
Hi @kalyann-vadada take a look at this: https://orkes.io/content/reference-docs/operators/switch

**Reply by kalyann-vadada** [reply]
Hey I am trying to run this in local. Can't understand why there is no execution part here in the code. Am I missing something? Is it something we need to build ?
